### PR TITLE
add consulting page

### PR DESCRIPTION
**Pull request by hamelsmu**
Add a consulting section.

@jxnl can you review please?   This is linked to my typeform, which I can send you.

### CODE REVIEW

File: services.qmd [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+We will save your engineering team time and money by steering them away from common pitfalls, selecting the best tools, and walking them through an approach to building AI products. Furthermore, we will introduce you to talent, vendors and partners to accelerate your goals.  
+
+### 2. Comprehensive Consulting
+
+Everything in strategic consulting, plus we build and deploy components of AI products for you.  We will work with your team to understand your needs, build a roadmap, and execute on it.
+
+## Pricing
+
+The cost of our services depends on the scope of the project.  However, the following are typical prices for our services:
```
Comment (jxnl) [inline services.qmd:30]: happy to say 

"lead by jason and hamel, with access to other experts in the field, from open source maintainers and high level operators that are building with ai"

File: services.qmd [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- Strategic Consulting: $18,750 / month
+- Comprehensive Consulting: $98,500 / month
```
Comment (jxnl) [inline services.qmd:33]: i wonder if we should extend it to say, 200k for minimum 3 month engagement

File: services.qmd [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+## Problems We Solve
+
+You should consider hiring us if:
+
+- You are hitting a plateau in getting your AI to do what you want.
+- Your LLMs need to be cheaper or faster.
+- You are overwhelmed by tools & frameworks.
+- You feel lost on what to prioritize and what experiments you should run.
```
[the remark this answers is not in the document]
Comment (eugeneyan) [reply services.qmd:13]: Do you also need to sell on the benefit?

- You need help prioritizing the roadmap and get to an MVP faster
- You need help building a tech team that becomes self-sufficient faster

File: services.qmd [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- You need outside expertise to evaluate your needs and vet potential talent.
+
+## Services
```
Comment (eugeneyan) [inline services.qmd:16]: Would it make sense to include a workshop that runs for a few days to teach the team the essentials of evals, RAG, recommended tools, etc. Essentially stuff that you already have in the course.

Here's a consultant that conducts a [three-day positioning workshop for 65-80k a pop](https://www.aprildunford.com/consultant).

**Review by eugeneyan — APPROVED** [review]
left a few comments; no shipblockers

**Pull request merged**